app/structure/every-fucking-thing-factory.js

```diff
@@ -29,9 +30,21 @@ export default class EveryFuckingThingFactory {
     }
 
     createGame(io) {
-        const players = this.createPlayers();
-        const snakes = this.getPlayersSnakes(players);
-        return new Game(this.createBoard(snakes), players, TIMEOUT, io);
+        return new Promise(resolve => {
+            this.playersAlgorithms = [];
+
+            SnakeLoader.importAll().then(imports => {
```

Comment:
Creating new function `getSnakeAlgoritms` will be definitely better

Reply:
👍